Tools/jit/_stencils.py

```diff
@@ -348,9 +325,21 @@ def _emit_global_offset_table(self) -> None:
             )
             self.data.body.extend([0] * 8)
 
+    def _get_trampoline_mask(self) -> str:
+        bitmask: int = 0
+        trampoline_mask: list[str] = []
+        for ordinal in self.trampolines:
+            bitmask |= 1 << ordinal
+        if bitmask:
+            trampoline_mask = [
+                f"0x{(bitmask >> i*32) & ((1 << 32) - 1):x}"
+                for i in range(0, SYMBOL_MASK_SIZE)
+            ]
```

Comment:
I had a bit of trouble following this, so I tried rewriting it. What do you think? This way we don't need to hardcode `SYMBOL_MASK_SIZE`, and can just delete it.

```suggestion
        while bitmask:
            word = bitmask & ((1 << 32) - 1)
            trampoline_mask.append(f"{word:#04x}")
            bitmask >>= 32
```

Reply:
Done.